klam-ssh/v2/authorizedkeys_command.sh

```diff
@@ -15,7 +15,10 @@ else
   chown -R $1. /home/$1 > /dev/null
 fi
 
+echo "adding user to docker group"
+gpasswd -a $1 docker
+
 echo "Running authorizedkeys_command for $1" | systemd-cat -p info -t klam-ssh
 
-docker run --net=host --rm -e ROLE_NAME=${ROLE_NAME} -e ENCRYPTION_ID=${ENCRYPTION_ID} -e ENCRYPTION_KEY=${ENCRYPTION_KEY} -e KEY_LOCATION_PREFIX=${KEY_LOCATION_PREFIX} ${IMAGE} /usr/lib/klam/getKeys.py $1
+docker run --privileged --userns=host --rm -e ROLE_NAME=${ROLE_NAME} -e ENCRYPTION_ID=${ENCRYPTION_ID} -e ENCRYPTION_KEY=${ENCRYPTION_KEY} -e KEY_LOCATION_PREFIX=${KEY_LOCATION_PREFIX} ${IMAGE} /usr/lib/klam/getKeys.py $1
```

Comment:
While this does work when namespaces are enabled, we also have to support when it is not. This should be set as a variable instead.